modules/ROOT/pages/websockets/websockets-connector-get-started.adoc

```diff
@@ -0,0 +1,133 @@
+* On Inbound Connection: will initiate a flow when a new connection to the Inbound Listener is established.
+* Outbound Message Listener: exposes an endpoint that will trigger a flow when an Outbound WebSocket receives a message from an external system.
+* On Close: will trigger the containing flow when a WebSocket (either Inbound or Outbound) is closed.
+
+== Operations
+
+* Open Outbound Socket: Connects to an external system and produces an outbound WebSocket.
+* Send: sends a message through a WebSocket.
+* Subscribe Groups: subscribes a WebSocket to one or more groups. It can be invoked on the same socket as many times as necessary, each invocation being additive over the previous ones. Repeated groups will be ignored.
+* Unsubscribe Groups: unsubscribes a WebSocket from one or more groups. It can be invoked on the same socket as many times as necessary, each invocation being additive over the previous ones. Repeated groups will be ignored.
+* Broadcast: send a message to several sockets classified by path, type and group.
+* Close: closes a WebSocket connection.
+
+== Security Schemes
+
+The connector supports securing both the client and server endpoints through the use of TLS. This means that the following URI schemes are supported:
+
+* WS: identifies a WebSocket server and resource name without any encryption (e.g: ws://localhost:80).
+* WSS: identifies a WebSocket server and resource name, secure through TLS (e.g: wss://localhost:433).
+
+== About Connectors
+
+Anypoint connectors are Mule runtime engine extensions that enable you to connect 
+to APIs and resources on external systems, such as Salesforce, WebSocket, ServiceNow,
+and Twitter.
+
+== Prerequisites
+
+Before creating an app, you must have access to the WebSocket target resource and 
+Anypoint Platform. You must also understand how to create a Mule app using 
+Design Center or Anypoint Studio.
+
+== Audience
+
+* Starting user -- Read this Get Started document and create your Mule app in 
+xref:websockets/websocket-connector-design-center.adoc[Design Center] or 
+xref:websockets/websocket-connector-studio.adoc[Anypoint Studio]. 
+The xref:websockets/websocket-connector-config-topics.adoc[Additional Configuration Information] topic
+helps you extend your understanding of WebSocket actions.
+* Power user - Read xref:websockets/websocket-connector-xml-maven.adoc[XML and Maven Support], 
+the xref:websockets/websocket-connector-config-topics.adoc[Additional Configuration Information] topic, 
+and xref:websockets/websocket-connector-examples.adoc[Examples].
+
+=== Compatibility
+
+The WebSocket Connector is compatible with:
+
+|===
+|Application/Service|Version
+|Mule Runtime|4.2.2 or higher
+|Amazon Java SDK for EC2|v1.11.134
+|Java |1.8.0_x 
+|Anypoint Studio|7.4 or higher
+|===
+
+Audience without the "Additional Configuration Information" topic:
+
+== Audience
+
+* Starting user -- Read this Get Started document and create your Mule app in 
```

Comment:
this audience segregation doesn't apply here. This is a complex connector and all context is relevant.